lib/scidata/imdb_reviews.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Scidata.IMDBReviews do
+  @base_url "http://ai.stanford.edu/~amaas/data/sentiment/"
+  @dataset_file "aclImdb_v1.tar.gz"
+
+  alias Scidata.Utils
+
+  def download(opts \\ []) do
+    download_dataset(:train, opts)
+  end
+
+  def download_test(opts \\ []) do
+    download_dataset(:test, opts)
+  end
+
+  defp download_dataset(dataset_type, opts) do
+    transform_inputs = opts[:transform_inputs] || (& &1)
+    transform_labels = opts[:transform_labels] || (& &1)
+
+    files = Utils.get!(@base_url <> @dataset_file).body
+
+    pos_files =
+      Enum.filter(files, fn {fname, _} ->
+        file_match?(fname, dataset_type, :pos)
+      end)
+
+    neg_files =
+      Enum.filter(files, fn {fname, _} ->
+        file_match?(fname, dataset_type, :neg)
+      end)
+
+    :rand.seed(:exsss, {101, 102, 103})
```

Comment:
If we are hardcoding a seed, then shuffle is deterministic. Is that what we really want? Also, maybe we should let the users shuffle?

Also, should we let positive and negative be an argument, given they are different files? I think we should try to avoid doing passes over the data, as that can be expensive?